packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -789,6 +795,65 @@ const schema: SchemaType<"Tags"> = {
       return rel?.parentDocumentId;
     },
   }),
+
+  maxScore: resolverOnlyField({
+    type: Number,
+    canRead: ['guests'],
+    resolver: async (tag, args, context) => {
+      const { MultiDocuments } = context;
+
+      // Get the tag's own baseScore
+      const tagScore = tag.baseScore || 0;
+
+      // Get multidocuments' baseScores
+      const multiDocuments = await getWithLoader(
+        context,
+        MultiDocuments,
+        'multiDocuments',
```

Comment:
I would call this something more distinct to avoid potential future collisions

Reply:
(and it's probably worth pulling out the entire loader into a function so you can safely re-use it across two resolvers without needing to change things in both places if you do ever change anything)